### PR TITLE
docs: update guides and integrations

### LINKED ISSUE #3: Import guides from previous website
This issue tracks the activity of importing the guides from the previous website https://capsule.clastix.io.

### PR DESCRIPTION
**Pull request by maxgio92**
This PR imports the currently published guides on [capsule.clastix.io](https://capsule.clastix.io) website.

Fixes #3

### CODE REVIEW

**Comment by maxgio92** [comment]
@oliverbaehler PTAL :)

**Comment by maxgio92** [comment]
Blocked by #6

**Comment by oliverbaehler** [comment]
@maxgio92 Note, i have already ported most of the guides, just to a different structure:
https://projectcapsule.dev/docs/operating/

**Comment by maxgio92** [comment]
So @oliverbaehler what's the status? Has part of what this PR achieves in the meantime superseded?

What's the scope of guides if guides are moved under the Operating section? I'm not so sure about including integration with 3rd party projects under Operating.

Can we please coordinate in the dedicated issue #3? @Svarrogh1337 did a great job summarizing the status of the mirgation.

**Comment by maxgio92** [comment]
Hi @oliverbaehler, I've just updated the PR accordingly. PTAL :-)
Thank you

File: content/en/integrations/tools/dashboard.md [hunk 1]
```diff
@@ -70,6 +70,31 @@ helm repo add oauth2-proxy https://oauth2-proxy.github.io/manifests
 helm install oauth2-proxy oauth2-proxy/oauth2-proxy -n ${KUBERNETES_DASHBOARD_NAMESPACE} -f values-oauth2-proxy.yaml
 ```
 
+## Configuring an Keycloak
```
[the remark this answers is not in the document]
Comment (maxgio92) [reply content/en/integrations/tools/dashboard.md]: Thanks

**Review by prometherion — APPROVED** [review]

**Pull request merged**